lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -3108,6 +3121,14 @@ IRBuilderAsmJs::BuildLong1Int1(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::Reg
         srcOpnd = BuildSrcOpnd(srcRegSlot, TyUint32);
```

Comment:
> Js::OpCodeAsmJs::Conv_ITD & Js::OpCodeAsmJs::Conv_UTD

This looks like a bad copy paste on my part, could you remove it while you're there